stacks/nodejs-svelte/devfile.yaml

```diff
@@ -25,20 +25,20 @@ components:
       endpoints:
         - name: http
           targetPort: 5000
-      image: node:lts-slim
+      image: rhel8/nodejs-16:latest
```

Comment:
I think you meant `registry.redhat.io/rhel8/nodejs-16`. Images without a registry server specified _generally_ default to `docker.io` on most Kubernetes distributions

Either way, we can't use this image. Images on `registry.redhat.io` require licenses (and therefore authentication) to pull, which means we can't use it.

Reply:
Yeah I just realized that I used RHEL8 instead of UBI8